From 43e8dfacb45e470d02d3880ceb28a4494262f4c9 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Florian=20M=C3=BCllner?= <fmuellner@gnome.org>
Date: Wed, 18 Jul 2018 23:03:30 +0200
Subject: [PATCH] build: Don't use libdir for systemd unit path

Some systems define it to /usr/lib64 on 64-bit systems, which is not
where systemd looks for unit files. Just hardcode 'lib' in the install
prefix until we get to use the pkg-config file.

https://gitlab.gnome.org/GNOME/gnome-shell/issues/424
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 7e5fe3832..65f6c7f1f 100644
--- a/meson.build
+++ b/meson.build
@@ -64,7 +64,7 @@ plugindir = get_variable('BROWSER_PLUGIN_DIR', mozplugindir)
 # systemduserunitdir = systemd_dep.get_pkgconfig_variable('systemduserunitdir',
 #                                                         define_variable: ['prefix', prefix])
 # and uncomment systemd_dep below
-systemduserunitdir = join_paths(libdir, 'systemd', 'user')
+systemduserunitdir = join_paths(prefix, 'lib', 'systemd', 'user')
 
 keybindings_dep = dependency('gnome-keybindings', required: false)
 if keybindings_dep.found()
-- 
2.17.1

